app/controllers/EnforcementEngineResponseProcessor.scala

```diff
@@ -0,0 +1,57 @@
+package controllers
+
+import controllers.EnforcementEngineResponseProcessor.debugMode
+import play.api.libs.json.{JsError, JsSuccess, JsValue, Json}
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.slf4j.LoggerFactory
+import play.api.Configuration
+
+import scala.collection.mutable.Stack
+
+object EnforcementEngineResponseProcessor {
+  private val LOGGER = LoggerFactory.getLogger("ProcessEnforcementEngineResponse")
+  var response : String = ""
+  var query: String = ""
+  var tableNames: Stack[String] = new Stack[String]()
+  var queryOnTables: String = ""
+  var debugMode: Boolean = false
+
+  def processResponse (spark: SparkSession, config: Configuration, response:String, debugMode:Boolean, dfShowLen: Int): DataFrame = {
+    this.debugMode = debugMode
+    val json: JsValue = Json.parse(response)
```

Comment:
Should be changed to normal object, as discussed in the policy-enforcement review.